src/odemis/model/test/futures_test.py

```diff
@@ -1,25 +1,31 @@
 # -*- coding: utf-8 -*-
```

Comment:
Not your fault but I think this file is missing the shebang while it should have one:

> #!/usr/bin/env python3

Reply:
Should it be there in every file? I found a few more where we do not have it.